website/server/middlewares/ensureAccessRight.js

```diff
@@ -16,7 +16,7 @@ export function ensureSudo (req, res, next) {
   let user = res.locals.user;
 
   if (!user.contributor.sudo) {
-    return next(new NotAuthorized(res.t('noSudoAccess')));
+    return next(new NotAuthorized('You don\'t have sudo access.'));
```

Comment:
Instead of using plain strings could you use `apiMessages`, ( see https://github.com/HabitRPG/habitica/blob/develop/website/server/libs/apiMessages.js and https://github.com/HabitRPG/habitica/search?utf8=%E2%9C%93&q=apimessages&type=) so that the string is not translated but not a plain string either? Thanks and sorry for the delay in reviewing this!